examples/data_types_and_io/data_types_and_io/attribute_access.py

```diff
@@ -0,0 +1,158 @@
+# %% [markdown]
+# (attribute_access)=
+#
+# # Attribute Access
+#
+# ```{eval-rst}
+# .. tags:: Basic
+# ```
+#
+# Flyte allows users to access attributes directly on output promises for List, Dict, Dataclass, and combinations of them. This allows users to pass attributes of the output directly in workflows, making it more convenient to work with complex data structures.
```

Comment:
The first sentence would read a little more smoothly as "Flyte allows users to access attributes directly on output promises for List, Dict, Dataclass, and combinations of these types" or "combinations thereof".